cmd/raw.go

```diff
@@ -342,20 +297,15 @@ func createConnectionStepCmd() *cobra.Command {
 				return err
 			}
 
-			msgs, err := chains[src].CreateConnectionStep(chains[dst])
-			if err != nil {
-				return err
-			}
-
-			if len(msgs.Src) > 0 {
-				if err = sendAndPrint(msgs.Src, chains[src], cmd); err != nil {
-					return err
-				}
-			} else if len(msgs.Dst) > 0 {
-				if err = sendAndPrint(msgs.Dst, chains[dst], cmd); err != nil {
+			_, _, modified, err := relayer.ExecuteConnectionStep(chains[src], chains[dst])
+			if modified {
+				if err := overWriteConfig(cmd, config); err != nil {
 					return err
 				}
 			}
+			if err != nil {
```

Comment:
nit: these should be swapped

Reply:
Will deal with it in future PR

Reply:
the issue here is open init could succeed and open try could fail. In this case we want to update the open init id instead of creating a new id upon retry